geode-benchmarks/src/main/java/org/apache/geode/benchmark/tests/ClientServerTopology/parameters/JVMParamters.java

```diff
@@ -12,10 +12,9 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.benchmark.parameters;
-
-public class JVMParameters {
+package org.apache.geode.benchmark.tests.ClientServerTopology.parameters;
```

Comment:
Camel case package name needs a bit weird. Does the JVMParameters class even need to be a it's own package? seems like it shouldn't be under client/server